src/main/java/org/reactome/release/qa/check/HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntitiesCheck.java

```diff
@@ -16,13 +18,13 @@
 @SliceQACheck
 public class HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntitiesCheck extends AbstractQACheck {
 
-    private static final String INNATE_IMMUNITY_PATHWAY_DBID = "168249";
     private List<String> skiplistDbIds = new ArrayList<>();
+    private static final String skiplist = "resources/human_reactions_without_disease_and_have_nonhuman_physicalentities.txt";
 
     @Override
     public QAReport executeQACheck() throws Exception {
         QAReport report = new QAReport();
-        this.skiplistDbIds.add(INNATE_IMMUNITY_PATHWAY_DBID);
+        this.skiplistDbIds.addAll(Files.readAllLines(Paths.get(skiplist)));
```

Comment:
Maybe it would be worth validating that the skip list contains only integers on each line (and throwing an exception flagging any offending data in the skip list file) before adding tto the `skiplistDbIds`?